docs/framework/windows-workflow-foundation/how-to-enable-persistence-for-workflows-and-workflow-services.md

```diff
@@ -52,10 +52,10 @@ Persistence can be enabled using an application configuration file by adding the
     <behaviors>
       <serviceBehaviors>
         <behavior name="myBehavior">
-          <SqlWorkflowInstanceStore connectionString="Data Source=myDatabaseServer;Initial Catalog=myPersistenceDatabase">
+          <sqlWorkflowInstanceStore connectionString="Data Source=myDatabaseServer;Initial Catalog=myPersistenceDatabase" />
```

Comment:
I made the `s` lowercase as it's in https://docs.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/windows-workflow-foundation/sqlworkflowinstancestore

Reply:
My understanding is that they're probably case insensitive (I'd have to check) but change looks good.